jsonnet/kube-prometheus/main.libsonnet

```diff
@@ -8,6 +8,7 @@ local nodeExporter = import './components/node-exporter.libsonnet';
 local prometheusAdapter = import './components/prometheus-adapter.libsonnet';
 local prometheusOperator = import './components/prometheus-operator.libsonnet';
 local prometheus = import './components/prometheus.libsonnet';
+local rbac = import './components/rbac.libsonnet';
```

Comment:
RBAC is not a separate component and it is only a part of some component (like prometheus). For this reason I think moving it out of other components (like `prometheus.libsonnet`) only complicates UX instead of simplifying it.